### DISCUSSION TITLE
v1.10.0 constant restarting...

### DISCUSSION THREAD

**Discussion by romfordpele** [raspiblitz/raspiblitz#4226] [Q&A]
Hi this is a node I built a while ago (2020). Was running v1.7 for a loooong time and just updated to v1.10.0. 

It worked for a day maybe but then I had issues and had to reboot. It never came back and would not unmount/mount HDD.

I am wondering if something happened to the required specs for the drive between the versions 1.7 and 1.10? Any ideas gratefully received!

Logs below:

***************************************************************
* RASPIBLITZ DEBUG LOGS 
***************************************************************
blitzversion: 1.10.0
chainnetwork: bitcoin / test
 20:08:12 up 6 days, 23:44,  2 users,  load average: 2.75, 2.63, 2.56

*** FAILED SERVICES ***
list any servcies with problems: sudo systemctl list-units --failed
  UNIT              LOAD   ACTIVE SUB    DESCRIPTION
● bitcoind.service  loaded failed failed Bitcoin daemon on mainnet
● lnd.service       loaded failed failed Lightning Network Daemon
● logrotate.service loaded failed failed Rotate log files

LOAD   = Reflects whether the unit definition was properly loaded.
ACTIVE = The high-level unit activation state, i.e. generalization of SUB.
SUB    = The low-level unit activation state, values depend on unit type.
3 loaded units listed.

*** SETUPPHASE / BOOTSTRAP ***
see logs: cat /home/admin/raspiblitz.log
setupPhase--> done
state--> ready

*** BACKGROUNDSERVICE ***
to monitor Background service call: sudo journalctl -f -u background

*** BLOCKCHAIN (MAINNET) SYSTEMD STATUS ***
● bitcoind.service - Bitcoin daemon on mainnet
     Loaded: loaded (/etc/systemd/system/bitcoind.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Mon 2023-10-23 20:08:02 BST; 9s ago
    Process: 3076302 ExecStartPre=/home/admin/config.scripts/bitcoin.check.sh prestart mainnet (code=exited, status=0/SUCCESS)
    Process: 3076336 ExecStart=/usr/local/bin/bitcoind -daemonwait -conf=/mnt/hdd/bitcoin/bitcoin.conf -datadir=/mnt/hdd/bitcoin (code=exited, status=1/FAILURE)
        CPU: 257ms

Oct 23 20:08:02 raspberrypi systemd[1]: bitcoind.service: Failed with result 'exit-code'.
Oct 23 20:08:02 raspberrypi systemd[1]: Failed to start Bitcoin daemon on mainnet.

*** LAST BLOCKCHAIN (MAINNET) ERROR LOGS ***
sudo journalctl -u bitcoind -b --no-pager -n20
-- Journal begins at Mon 2023-10-23 10:35:00 BST, ends at Mon 2023-10-23 20:08:12 BST. --
Oct 23 20:08:00 raspberrypi systemd[1]: bitcoind.service: Failed with result 'exit-code'.
Oct 23 20:08:00 raspberrypi systemd[1]: Failed to start Bitcoin daemon on mainnet.
Oct 23 20:08:00 raspberrypi systemd[1]: bitcoind.service: Scheduled restart job, restart counter is at 3.
Oct 23 20:08:00 raspberrypi systemd[1]: Stopped Bitcoin daemon on mainnet.
Oct 23 20:08:01 raspberrypi systemd[1]: Starting Bitcoin daemon on mainnet...
Oct 23 20:08:01 raspberrypi systemd[1]: bitcoind.service: Control process exited, code=exited, status=1/FAILURE
Oct 23 20:08:01 raspberrypi systemd[1]: bitcoind.service: Failed with result 'exit-code'.
Oct 23 20:08:01 raspberrypi systemd[1]: Failed to start Bitcoin daemon on mainnet.
Oct 23 20:08:01 raspberrypi systemd[1]: bitcoind.service: Scheduled restart job, restart counter is at 4.
Oct 23 20:08:01 raspberrypi systemd[1]: Stopped Bitcoin daemon on mainnet.
Oct 23 20:08:01 raspberrypi systemd[1]: Starting Bitcoin daemon on mainnet...
Oct 23 20:08:02 raspberrypi bitcoind[3076336]: Error: Error reading configuration file: specified config file "/mnt/hdd/bitcoin/bitcoin.conf" could not be opened.
Oct 23 20:08:02 raspberrypi systemd[1]: bitcoind.service: Control process exited, code=exited, status=1/FAILURE
Oct 23 20:08:02 raspberrypi systemd[1]: bitcoind.service: Failed with result 'exit-code'.
Oct 23 20:08:02 raspberrypi systemd[1]: Failed to start Bitcoin daemon on mainnet.
Oct 23 20:08:02 raspberrypi systemd[1]: bitcoind.service: Scheduled restart job, restart counter is at 5.
Oct 23 20:08:02 raspberrypi systemd[1]: Stopped Bitcoin daemon on mainnet.
Oct 23 20:08:02 raspberrypi systemd[1]: bitcoind.service: Start request repeated too quickly.
Oct 23 20:08:02 raspberrypi systemd[1]: bitcoind.service: Failed with result 'exit-code'.
Oct 23 20:08:02 raspberrypi systemd[1]: Failed to start Bitcoin daemon on mainnet.

*** LAST BLOCKCHAIN (MAINNET) INFO LOGS ***
sudo tail -n 50 /mnt/hdd/bitcoin/debug.log

*** LND (MAINNET) SYSTEMD STATUS ***
- OFF by config -

*** CORE LIGHTNING (MAINNET) SYSTEMD STATUS ***
- not activated -

*** BLOCKCHAIN (TESTNET) SYSTEMD STATUS ***
- OFF by config -
*** LND (TESTNET) SYSTEMD STATUS ***
- OFF by config -

*** CORE LIGHTNING (TESTNET) SYSTEMD STATUS ***
- not activated -

*** BLOCKCHAIN (SIGNET) SYSTEMD STATUS ***
- OFF by config -
*** LND (SIGNET) SYSTEMD STATUS ***
- OFF by config -

*** CORE LIGHTNING (SIGNET) SYSTEMD STATUS ***
- not activated -

*** NGINX SYSTEMD STATUS ***
● nginx.service - A high performance web server and a reverse proxy server
     Loaded: loaded (/lib/systemd/system/nginx.service; enabled; vendor preset: enabled)
    Drop-In: /etc/systemd/system/nginx.service.d
             └─raspiblitz.conf
     Active: active (running) since Mon 2023-10-16 20:18:08 BST; 6 days ago
       Docs: man:nginx(8)
   Main PID: 939 (nginx)
      Tasks: 5 (limit: 4531)
        CPU: 211ms
     CGroup: /system.slice/nginx.service
             ├─939 nginx: master process /usr/sbin/nginx -g daemon on; master_process on;
             ├─940 nginx: worker process
             ├─941 nginx: worker process
             ├─942 nginx: worker process
             └─943 nginx: worker process

Warning: journal has been rotated since unit was started, output may be incomplete.

*** LAST NGINX LOGS ***
sudo journalctl -u nginx -b --no-pager -n20
-- Journal begins at Mon 2023-10-23 10:35:00 BST, ends at Mon 2023-10-23 20:08:12 BST. --
-- No entries --
--> CHECK CONFIG: sudo nginx -t
nginx: [emerg] cannot load certificate "/mnt/hdd/app-data/nginx/tls.cert": BIO_new_file() failed (SSL: error:02001005:system library:fopen:Input/output error:fopen('/mnt/hdd/app-data/nginx/tls.cert','r') error:2006D002:BIO routines:BIO_new_file:system lib)
nginx: configuration file /etc/nginx/nginx.conf test failed

*** BLITZAPI STATUS ***
installed=1
repo='https://github.com/fusion44/blitz_api.git'
branch='blitz-v1.10'
commit='191c77b4d4cc5c9d97dc50da31e5cfb2e5bcb930'
● blitzapi.service - BlitzBackendAPI
     Loaded: loaded (/etc/systemd/system/blitzapi.service; enabled; vendor preset: enabled)
     Active: active (running) since Mon 2023-10-16 20:18:07 BST; 6 days ago
   Main PID: 818 (python)
      Tasks: 13 (limit: 4531)
        CPU: 3h 24min 12.174s
     CGroup: /system.slice/blitzapi.service
             └─818 /home/blitzapi/blitz_api/venv/bin/python -m uvicorn app.main:app --port 11111 --host=*.*.*.* --root-path /api

Oct 23 20:08:11 raspberrypi python[818]: 2023-10-23 20:08:11 | ⚠️ | utils.py:69 | Key 'hdd_capacity_bytes' not found in Redis DB.
Oct 23 20:08:12 raspberrypi python[818]: 2023-10-23 20:08:12 | ❌ | service.py:194 | Aiohttp client connection error: Cannot connect to host *.*.*.*:8332 ssl:default [Connect call failed ('127.0.0.1', 8332)]

*** LAST BLITZAPI LOGS ***
sudo journalctl -u blitzapi -b --no-pager -n20
-- Journal begins at Mon 2023-10-23 10:35:00 BST, ends at Mon 2023-10-23 20:08:13 BST. --
Oct 23 20:07:59 raspberrypi python[818]: 2023-10-23 20:07:59 | ⚠️ | utils.py:69 | Key 'hdd_capacity_bytes' not found in Redis DB.
Oct 23 20:08:01 raspberrypi python[818]: 2023-10-23 20:08:00 | ❌ | service.py:194 | Aiohttp client connection error: Cannot connect to host *.*.*.*:8332 ssl:default [Connect call failed ('127.0.0.1', 8332)]
Oct 23 20:08:01 raspberrypi python[818]: 2023-10-23 20:08:01 | ℹ️ | utils.py:67 | Key 'tor_web_addr' not found in Redis DB.
Oct 23 20:08:01 raspberrypi python[818]: 2023-10-23 20:08:01 | ⚠️ | utils.py:69 | Key 'hdd_capacity_bytes' not found in Redis DB.
Oct 23 20:08:02 raspberrypi python[818]: 2023-10-23 20:08:02 | ❌ | service.py:194 | Aiohttp client connection error: Cannot connect to host *.*.*.*:8332 ssl:default [Connect call failed ('127.0.0.1', 8332)]
Oct 23 20:08:03 raspberrypi python[818]: 2023-10-23 20:08:03 | ℹ️ | utils.py:67 | Key 'tor_web_addr' not found in Redis DB.
Oct 23 20:08:03 raspberrypi python[818]: 2023-10-23 20:08:03 | ⚠️ | utils.py:69 | Key 'hdd_capacity_bytes' not found in Redis DB.
Oct 23 20:08:04 raspberrypi python[818]: 2023-10-23 20:08:04 | ❌ | service.py:194 | Aiohttp client connection error: Cannot connect to host *.*.*.*:8332 ssl:default [Connect call failed ('127.0.0.1', 8332)]
Oct 23 20:08:07 raspberrypi python[818]: 2023-10-23 20:08:05 | ℹ️ | utils.py:67 | Key 'tor_web_addr' not found in Redis DB.
Oct 23 20:08:07 raspberrypi python[818]: 2023-10-23 20:08:05 | ⚠️ | utils.py:69 | Key 'hdd_capacity_bytes' not found in Redis DB.
Oct 23 20:08:07 raspberrypi python[818]: 2023-10-23 20:08:06 | ❌ | service.py:194 | Aiohttp client connection error: Cannot connect to host *.*.*.*:8332 ssl:default [Connect call failed ('127.0.0.1', 8332)]
Oct 23 20:08:07 raspberrypi python[818]: 2023-10-23 20:08:07 | ℹ️ | utils.py:67 | Key 'tor_web_addr' not found in Redis DB.
Oct 23 20:08:07 raspberrypi python[818]: 2023-10-23 20:08:07 | ⚠️ | utils.py:69 | Key 'hdd_capacity_bytes' not found in Redis DB.
Oct 23 20:08:08 raspberrypi python[818]: 2023-10-23 20:08:08 | ❌ | service.py:194 | Aiohttp client connection error: Cannot connect to host *.*.*.*:8332 ssl:default [Connect call failed ('127.0.0.1', 8332)]
Oct 23 20:08:09 raspberrypi python[818]: 2023-10-23 20:08:09 | ℹ️ | utils.py:67 | Key 'tor_web_addr' not found in Redis DB.
Oct 23 20:08:09 raspberrypi python[818]: 2023-10-23 20:08:09 | ⚠️ | utils.py:69 | Key 'hdd_capacity_bytes' not found in Redis DB.
Oct 23 20:08:10 raspberrypi python[818]: 2023-10-23 20:08:10 | ❌ | service.py:194 | Aiohttp client connection error: Cannot connect to host *.*.*.*:8332 ssl:default [Connect call failed ('127.0.0.1', 8332)]
Oct 23 20:08:11 raspberrypi python[818]: 2023-10-23 20:08:11 | ℹ️ | utils.py:67 | Key 'tor_web_addr' not found in Redis DB.
Oct 23 20:08:11 raspberrypi python[818]: 2023-10-23 20:08:11 | ⚠️ | utils.py:69 | Key 'hdd_capacity_bytes' not found in Redis DB.
Oct 23 20:08:12 raspberrypi python[818]: 2023-10-23 20:08:12 | ❌ | service.py:194 | Aiohttp client connection error: Cannot connect to host *.*.*.*:8332 ssl:default [Connect call failed ('127.0.0.1', 8332)]

*** BLITZ WebUI STATUS ***
installed=1
repo='https://github.com/raspiblitz/raspiblitz-web.git'
branch='release/v1.10'
commit='64de1160e5a65ca179b8e41cd0ba1348539dff34'

- TOUCHSCREEN is OFF by config
- Loop is OFF by config
- LND-RTL is OFF by config
- CL-RTL is OFF by config
- Electrum Rust Server is OFF by config
- LIT is OFF by config
- LNDg is OFF by config
* BTCPayServer is OFF by config
* BTC-RPC-Explorer is OFF by config
- LNbits is OFF by config
- Thunderhub is OFF by config
- SPECTER is OFF by config
- SPHINX is OFF by config
- FINTS is OFF by config

*** MOUNTED DRIVES ***
df -T -h
Filesystem     Type      Size  Used Avail Use% Mounted on
/dev/root      ext4       30G   16G   13G  57% /
devtmpfs       devtmpfs  1.9G     0  1.9G   0% /dev
tmpfs          tmpfs     1.9G   16K  1.9G   1% /dev/shm
tmpfs          tmpfs     769M  1.0M  768M   1% /run
tmpfs          tmpfs     5.0M     0  5.0M   0% /run/lock
tmpfs          tmpfs      32M   12K   32M   1% /var/cache/raspiblitz
/dev/mmcblk0p1 vfat      255M   32M  224M  13% /boot
/dev/sda1      ext4      916G  604G  266G  70% /mnt/hdd
tmpfs          tmpfs     385M   16K  385M   1% /run/user/1000
tmpfs          tmpfs     385M   16K  385M   1% /run/user/117

*** SD CARD HOMES ***
sudo du -ahd1 /home
1.4G	/home/mempool
891M	/home/bitcoin
35M	/home/pi
2.4G	/home/btcpay
430M	/home/admin
180M	/home/btcrpcexplorer
450M	/home/rtl
586M	/home/joinmarket
565M	/home/jam
1.6G	/home/thunderhub
663M	/home/lnbits
145M	/home/specter
816M	/home/blitzapi
9.9G	/home

*** LOGFILES ***
Archived and active journals take up 238.0M in the file system.
1.1G	/var/log

*** DATADRIVE ***
# RASPIBLITZ DATA DRIVE Status

# BASICS
isMounted=1
isBTRFS=0
hddRaspiData=0
hddRaspiVersion=''
isSMART=0
isSSD=0
datadisk='sda'
datapartition='sda1'
hddCandidate='sda'
hddPartitionCandidate='sda1'
hddBlocksBitcoin=0
hddBytes=
hddGigaBytes=
hddUsedInfo='16G
16K
32M
604G (5711370%)'
error='unkown command'

*** NETWORK ***
localip=*.*.*.*
localiprange=*.*.*.*/24
dhcp=1
network_device=eth0

*** ZRAM ***
downloaded=1
serviceLoaded=1
serviceActive=1

*** HARDWARE TEST RESULTS ***
UndervoltageReports in Logs: 0

*** SYSTEM CACHE STATUS ***
system_board="rp4"
system_count_longscan="2"
system_ram_mb="3844"
system_ups_battery=""
system_temp_fahrenheit="136"
system_count_undervoltage="0"
system_up="603934"
system_count_start_lightning="1"
system_ups_status="OFF"
system_vm_vagrant="0"
system_ram_available_mb="3062"
system_cpu_load=" 4.19, 3.25, 2.80"
system_init_time="1697483893"
system_ram_gb="4"
system_temp_celsius="58"
system_count_start_blockchain="90326"
ln_default_online="0"
ln_default_error_full=""
ln_default_version="0.16.4-beta"
ln_default_activated="1"
ln_default_ready="0"
ln_default_running="0"
ln_default_error_short=""
ln_default_locked="0"
ln_default_sync_initial_done="1"
btc_default_error_short=""
btc_default_version="v25.0.0"
btc_default_running="0"
btc_default_blocks_behind="78"
btc_default_peers="2"
btc_default_activated="1"
btc_default_sync_initial_done="0"
btc_default_synced="0"
btc_default_error_full=""
btc_default_ready="0"
btc_default_sync_initialblockdownload="1"
btc_default_sync_progress="0.9980679252314205"
btc_default_mempool_transactions="0"
btc_default_sync_percentage="99.81"
btc_default_blocks_data_kb="575222624"
btc_default_port="8333"
btc_default_blocks_headers="811739"
btc_default_online="0"
btc_default_blocks_verified="811661"
*** POSSIBLE ERROR REPORTS ***

**Comment by rootzoll** [comment]
This seems to the the problem:
`Oct 23 20:08:02 raspberrypi bitcoind[3076336]: Error: Error reading configuration file: specified config file "/mnt/hdd/bitcoin/bitcoin.conf" could not be opened.`

Not sure why this happens but this should fix itself when you recover with a fresh sd card image of v1.10.0
